content/pages/housing-assistance/adaptive-housing-grants.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: page-breadcrumbs.html
+title: Adaptive Housing Grants
+concurrence: 
+template: 1-topic-landing
+relatedlinks:
+  - url: /housing-assistance/home-loans/
+    title: Home Loans
+    description: Choose from one of the VA-backed loan options and find out how to apply for a loan to buy, improve, or refinance a home.
+---
+
+<div class="va-introtext">
+
+We offer grants to severely disabled Servicemembers and Veterans to buy or change a home to meet their needs and live more independently. Changing a home might involve installing ramps or widening doorways. If you’re a disabled Servicemember or Veteran, find out if you can get a grant to help meet your housing needs.
+
+</div>
+
+<div class="feature" markdown=“1”>
+
+### Can I get a Specially Adapted Housing (SAH) grant—and how much funding does this grant offer? 
+
+You may be able to get an SAH grant if you’re using the grant money to buy, build, or change your permanent home (a home you plan to live in for a long time) if:
+
+-	You own or will own the home, **and** 
```

Comment:
@bethpotts this is one of those formatting errors Rachael mentioned from copy/pasting from Word - even though when you edit the file the bullet and spacing looks correct, you can see here in the code output that there's a bunch of extra space between the bullet and the text. That unfortunately causes the sub-bullets to not indent correctly.

I'll fix this here, but hopefully this helps with troubleshooting - when things are looking wonky, try deleting and retyping the bullets or numbers in a list.